mathesar_ui/src/systems/table-view/table-inspector/column/ColumnMode.svelte

```diff
@@ -85,19 +86,27 @@
       </Collapsible>
     {/if}
 
-    <!-- Coming in the PR for chaging column data types -->
-    <!-- {#if column}
+    {#if column && !column.column.default?.is_dynamic}
       <Collapsible isOpen>
         <CollapsibleHeader
           slot="header"
           title="Default Value"
-          isDBLevelConfiguration
+          isDbLevelConfiguration
         />
         <div slot="content" class="content-container">
-          <SetDefaultValue />
+          <SetDefaultValue {column} />
+        </div>
+      </Collapsible>
+    {/if}
+
+    {#if column}
+      <Collapsible>
```

Comment:
The `Collapsible` components here need a triggerAppearance='plain' prop inorder to match the figma design. Notice that there are additional borders for these two collapsibles.

Reply:
33e7b2e2